docs/develop/01_sdk/02_cookbook/01_dids/04_full_did_query.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: did-query
-title: Resolve a DID
+title: Resolve a Full DID
```

Comment:
Why this change? The API is the same for both.

Reply:
Since the file is called `full_did_query` i assumed it was about full DIDs. If it's about light and full i would rename the file and change the title.

Reply:
Yes, I would say it's better to have a "resolve a DID" everywhere, as users should not really know what the difference is.

Reply:
[1dfe4c3](https://github.com/KILTprotocol/docs/pull/183/commits/1dfe4c30677266b1921b675d98b6c43606e27959)